nexus/db-queries/src/db/datastore/silo.rs

```diff
@@ -61,13 +62,32 @@ impl DataStore {
         debug!(opctx.log, "attempting to create built-in silos");
 
         use db::schema::silo::dsl;
-        let count = diesel::insert_into(dsl::silo)
-            .values([&*DEFAULT_SILO, &*INTERNAL_SILO])
-            .on_conflict(dsl::id)
-            .do_nothing()
-            .execute_async(&*self.pool_connection_authorized(opctx).await?)
+        use db::schema::silo_quotas::dsl as quotas_dsl;
+        let count = self
+            .pool_connection_authorized(opctx)
+            .await?
+            .transaction_async(|conn| async move {
+                diesel::insert_into(quotas_dsl::silo_quotas)
+                    .values(SiloQuotas::arbitrarily_high_default(
+                        DEFAULT_SILO.id(),
```

Comment:
Could use a comment on the possibly-surprising-in-the-future fact that we do this for default silo but not internal silo.